.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should prefer `.eslintrc.json` as the file name. "raw" `.eslintrc` files are deprecated.